content/en/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md

```diff
@@ -310,6 +310,27 @@ include it.
 In addition to enabling the feature in the cluster, a resource driver also has to
 be installed. Please refer to the driver's documentation for details.
 
+## Devices status
+
+{{< feature-state feature_gate_name="ResourceHealthStatus" >}}
+
+There are cases when devices fail or are shut down. The responsibility of the DRA plugin
+in this case is to notify the kubelet about the situation using the `WatchResources` API.
+
+Pods that were assigned to the failed devices will continue be assigned to this device.
+It is typical that code relying on the device will start failing and Pod may get
+into Failed phase if `restartPolicy` for the Pod was not `Always` or enter the crash loop
+otherwise.
```

Comment:
```suggestion
Code that relies on the devices will usually fail. Pods that don't set the `restartPolicy`
field to `Always` might enter the `Failed` phase. Pods that do set the `restartPolicy`
field to `Always` might enter a crash loop. 
```

Reply:
Merge this into the previous paragraph to make that a single "problem statement"